egs/wsj/asr1/conf/tuning/train_rnn.yaml

```diff
@@ -36,3 +36,6 @@ patience: 3
 
 # scheduled sampling option
 sampling-probability: 0.0
+
+# SpecAug
+use-specaug: True
```

Comment:
Can you update RESULTS.md if you wanna make this default

Reply:
Sure I will update code and RESULTS.md later.